deployment/.env.prod.example

```
@@ -0,0 +1,94 @@
+POSTGRES_PASSWORD=your-secure-db-password-here
+
+# Database port (optional, defaults to 5432)
+XYNE_DB_PORT=5432
+
+# =============================================================================
+# XYNE APPLICATION CONFIGURATION
+# =============================================================================
+
+# Keycloak integration
+KEYCLOAK_ENABLED=true
+KEYCLOAK_BASE_URL=http://localhost:8081
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The KEYCLOAK_BASE_URL key should go before the KEYCLOAK_ENABLED key

(UnorderedKey)